### PR TITLE
feat: list verification pipeline v1

### PR DESCRIPTION
**Pull request by matstyler**

### CODE REVIEW

File: src/schema/spectraTokens.schema.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        "type": "string"
+      },
+      "extensions": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "underlying": {
+            "type": "string",
+            "pattern": "^0x[a-fA-F0-9]{40}$"
+          },
```
Comment (Ulydev) [inline src/schema/spectraTokens.schema.json:44]:
```suggestion
          }, "externalLink": { "type": "string", "pattern": "https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)" },
```

Can we add this extension: `externalLink` 🙏

**Review by Ulydev — APPROVED** [review]
Looks good! Let's just add the property `externalLink` and LGTM :-)

File: src/schema/spectraTokens.schema.json (hunk 1 above)
Comment (matstyler) [reply src/schema/spectraTokens.schema.json:44]: we have the `externalLink` already (line 48) but I will extend it by the pattern

Comment (matstyler) [reply src/schema/spectraTokens.schema.json:44]: I had to change the regex and provided one doesn't work for our current value

**Pull request merged**